dev-tools/mage/crossbuild.go

```diff
@@ -202,11 +202,7 @@ func CrossBuildImage(platform string) (string, error) {
 	case platform == "darwin/arm64":
 		tagSuffix = "darwin-arm64-debian10"
 	case platform == "linux/arm64":
-		tagSuffix = "arm"
-		// when it runs on a ARM64 host/worker.
-		if runtime.GOARCH == "arm64" {
-			tagSuffix = "base-arm-debian9"
-		}
+		tagSuffix = "base-arm-debian9"
```

Comment:
I'm not sure about this change, is it coming from a backport?

Reply:
Yes, that was failing one of the CI stteps so I had to backport the change from https://github.com/elastic/beats/pull/42705/files#diff-df1a9f4fdcda58cbe426a97d94d780758159d6a8b9194d8f12544947a392e7ddR230.